src/scripts/settings/userSettings.js

```diff
@@ -253,6 +253,20 @@ export class UserSettings {
         return val !== 'false';
     }
 
+    /**
+     * Get or set 'Use Episode Images in Next Up' state.
+     * @param {boolean|undefined} val - Flag to enable 'Use Episode Images in Next Up' or undefined.
```

Comment:
Using `string|boolean|undefined` here _might_ make the SonarQube check pass.

Reply:
I prefer this:
```js
return this.get('useEpisodeImagesInNextUpAndResume', false) === 'true';
```